Makefile

```diff
@@ -151,6 +163,10 @@ config-generated.go:
 
 $(TARGET): $(SOURCES) $(GENERATED_FILES) Makefile | show-summary
 	$(QUIET_BUILD)go build -i -o $@ .
+	$(QUIET_BUILD)go build -i -ldflags "-X main.commit=${COMMIT} -X main.version=${VERSION} -X main.libExecDir=${LIBEXECDIR}" -o $@ .
+
+pause: $(PAUSEDIR)/$(PAUSEBIN).o
```

Comment:
Thanks for adding this target. But you need to add it to the `default` rule as it's not being built at `make` time for me and it would make sense if it was I think).